generate_ics.py

```diff
@@ -73,10 +81,17 @@ def main():
     now = datetime.datetime.now()
 
     # Get your timetable
-    with open('data.txt') as data_file:    
+    if not os.path.exists("data.txt"):
```

Comment:
Hey, this is a good check and it should be there, but it shouldn't be here. This should all happen alongwith the argument parsing.

The best things to do is parse the arguments, and set two variables `INPUT_FILE` and `OUTPUT_FILE`. Then, before starting the script, check if `INPUT_FILE` exists before starting execution.

Reply:
Done :+1: